metron-interface/metron-config/package.json

```diff
@@ -40,14 +40,14 @@
     "@types/node": "^10.9.4",
     "@types/tether": "^1.1.27",
     "ace-builds": "^1.2.5",
-    "bootstrap": "4.0.0-alpha.5",
+    "bootstrap": "^4.1.3",
     "core-js": "^2.5.7",
     "font-awesome": "^4.6.3",
     "jquery": "^3.3.1",
     "karma-phantomjs-launcher": "^1.0.4",
+    "popper.js": "^1.14.4",
```

Comment:
This is a new dependency for us, correct?  Can you comment on the project license, size of the community backing this, etc?  I am sure you have already considered these points, but we should have this discussion in the community for new dependencies.

Reply:
@nickwallen check out the contributor comments. It's MIT licensed and because bootstrap changed it's underlying library, not a choice @sardell had to make if my understanding is correct.

Reply:
Oops.  Way ahead of me!  Looks reasonable.